### ISSUE TITLE
Labelling Training Set - Bounding Boxes

### ISSUE THREAD

**Issue by GeorgePearse** [BinItAI/visdet#933]
**Original issue:** https://github.com/open-mmlab/mmdetection/issues/7823
**Created:** 2022-04-25
**Last updated:** 2022-04-27

---

Hi,

I have a question about training set labelling. Because I'm trying to detect an objects mask, I have been labelling my images as below:

![Screenshot from 2022-04-25 08-29-55](https://user-images.githubusercontent.com/85254524/165026256-fdf98d5e-cf88-45aa-8fe3-d321f5946bd2.png)

I only selected mask boundaries for mask detection and didn't select any bounding boxes. Do I need to select bounding boxes as well for more proper results? Moreover when I don't select bounding boxes, how the algorithm compares its results with the test set since there is no bounding box in the set?
